core/domain/classifier_services.py

```diff
@@ -118,3 +123,99 @@ def classify_string_classifier_rule(state, normalized_answer):
             return None
 
     return None
+
+
+def get_classifier_from_model(classifier_model):
+    """Gets a classifier domain object from a classifier model.
+
+    Args:
+        classifier_model: Classifier model instance in datastore.
+
+    Returns:
+        classifier: Domain object for the classifier.
+    """
+    return classifier_domain.Classifier(
+        classifier_model.id, classifier_model.exp_id,
+        classifier_model.exp_version_when_created,
+        classifier_model.state_name, classifier_model.algorithm_id,
+        classifier_model.cached_classifier_data,
+        classifier_model.data_schema_version)
+
+
+def get_classifier_by_id(classifier_id):
+    """Gets a classifier from a classifier id.
+
+    Args:
+        classifier_id: str. ID of the classifier.
+
+    Returns:
+        classifier: Domain object for the classifier.
+
+    Raises:
+        Exception: Entity for class ClassifierModel with id not found.
+    """
+    classifier_model = classifier_models.ClassifierModel.get(
+        classifier_id)
+    classifier = get_classifier_from_model(classifier_model)
+    return classifier
+
+
+def _create_classifier(classifier):
+    """Creates classifier model in the datastore given a classifier
+       domain object.
+
+    Args:
+        classifier: Domain object for the classifier.
+
+    """
+    classifier_id = classifier_models.ClassifierModel.create(
+        classifier.exp_id, classifier.exp_version_when_created,
+        classifier.state_name, classifier.algorithm_id,
+        classifier.cached_classifier_data, classifier.data_schema_version)
+    return classifier_id
+
+
+def _update_classifier(classifier_model, state_name):
+    """Updates classifier model in the datastore given a classifier
+    domain object.
+
+    Args:
+        classifier_model: Classifier model instance in datastore.
+        classifier: Domain object for the classifier.
+
+    Note: All of the properties of a classifier are immutable,
+    except for state_name.
```

Comment:
Since state_name is only mutable property, I agree with having a single method to change state name and avoid accidental modification of other properties.
@anmolshkl  what do you think?